controllers/dellcsivolumegroupsnapshot_controller_test.go

```diff
@@ -331,7 +320,7 @@ func (suite *VGSControllerTestSuite) TestVscNotFound() {
 	suite.runFakeVGManager(vgName, suite.mockUtils.Specs.Namespace, "VolumeSnapshotClass.snapshot.storage.k8s.io \"vsc\" not found")
 }
 
-// test getSourceVolIDs with with non-empty pvcLabel but no matching pvc. pvcList should be empty and return error
+// test getSourceVolIDs with non-empty pvcLabel but no matching pvc. pvcList should be empty and return error
 func (suite *VGSControllerTestSuite) TestNoMatchingPVC() {
```

Comment:
As I can see that we are using the `VGSControllerTestSuite` but at line no. 312, we are trying to remove the code that setup/mocks the required resources, is this fine?

Reply:
That code was actually trying to invoke the real client and due to that it was failing in actions as it is expecting KubeConfig. and hence removed it

Reply:
Got it, then in that case, make sense. Thank you!
Good to go from my end.